.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 80
```

Comment:
Please stick to max-line-length 80 as described in the contribution guideline.

Reply:
:(